poc/NaiveBayesClassifier/algorithm.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import GaussianNB, MultinomialNB
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.pipeline import Pipeline
+from sklearn.metrics import confusion_matrix
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns; sns.set()
+
+import sys
+
+if not (sys.version_info.major == 3 and sys.version_info.minor >= 5):
+    from sklearn.externals import joblib 
+    
+#Data computeed and processed.
+df = pd.read_csv('data.csv')
+df.drop(['Number', 'timestamp'], axis=1, inplace=True)
+
+input = df.drop('sleep', axis=1)
+target = df.sleep
+
+X_train, X_test, y_train, y_test = train_test_split(input, target, test_size=0.25, random_state=0)
+
+model = GaussianNB() 
+model.fit(X_train, y_train)
+
+##### Print model viability/score
+
+#print(model.score(X_test, y_test))
+#print(y_test[:10])
+#print(model.predict(X_test[:10]))
+
+##### Save the model as a pickle in a file
+#if not (sys.version_info.major == 3 and sys.version_info.minor >= 5):
+    #joblib.dump(model, 'modelSaved.pkl') 
```

Comment:
Why so many commented forgotten lines?

Reply:
As I have mentioned previously in the Readme.md file, there is an issue with saving your model as it asks your Python version to be downgraded (v:2.7). On the other hand, using confusion matrices in python 2.7 is not supported anymore. So, I had to make a choice for the following POC: Which was that, I am going to deliver the version with confusion matrix and Python 3+, where people with the wish to try the save/load feature will even be able to by removing comments, and obviously downgrading their python version (see Readme.MD). Would you like me, to edit/remove a bit for you?

@Leodau